src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ContainerService</PackageId>
+    <Description>Provides developers with a library to create and manage all Azure Kubernetes Service resources. </Description>
+    <Version>1.0.0</Version>
```

Comment:
Just to confirm, you are planning to start off with a stable version or are you planning initial preview versions before releasing a stable version?

Reply:
We are targeting the stable version at this time.